src/Domain/Creators/Sphere.cpp

```diff
@@ -47,6 +47,22 @@ Excision::Excision(
     std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
         local_boundary_condition)
     : boundary_condition(std::move(local_boundary_condition)) {}
+
+struct DistributionVisitor {
```

Comment:
Anonymous namespace.

Reply:
An anonymous namespace inside a detail namespace??

Reply:
I don't care if you leave it in the detail namespace or move it out.